tasks/deploy_get_stamp_from_route.yml

```diff
@@ -0,0 +1,14 @@
+---
+# Get deployment_stamp from a route
+
+- name: Get deployment_stamp of route[{{ app.name }}-{{ prefixe }}]
+  openshift_raw:
+    definition: "{{ lookup('template', 'templates/openshift/common/utils/get_deployment_stamp_form_route.yml.j2') | from_yaml }}"
+  register: my_route
```

Comment:
I am sure we can work on this variable name to improve its meaning. What about `app_route`?